### PR TITLE
docs(emptystate): add empty state documentation

### PR DESCRIPTION
**Pull request by samyak3009**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: docs/src/pages/components/emptyState/code.mdx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#### Live Demo
+<br/>
+
+<Preview name="ccomponents-emptystate-all--all" />
```
Comment (anuradha9712) [inline docs/src/pages/components/emptyState/code.mdx]: Please check the story url. I think there is some typo in `ccomponents`

Comment (samyak3009) [reply docs/src/pages/components/emptyState/code.mdx]: done

File: docs/src/pages/components/emptyState/usage.mdx [hunk 2]
```diff
@@ -0,0 +1,201 @@
+---
+title: Empty state
+description: The empty state indicates the absence of data and helps guide users on expected content and next steps.
+tabs: ['Usage', 'Code']
+showMobile: true
+keywords: ['Menu', 'Select', 'Listbox', 'Context menu']
```
Comment (anuradha9712) [inline docs/src/pages/components/emptyState/usage.mdx]: Please change this `keywords`. Keywords should be synonymous of the Components. Also `showMobile` must be `false` here as this component is not available in Mobile Design System

Comment (samyak3009) [reply docs/src/pages/components/emptyState/usage.mdx]: done

**Review by anuradha9712 — CHANGES_REQUESTED** [review]
Please rebase this branch with `develop` branch and upto 1 commit

File: docs/src/pages/components/emptyState/usage.mdx (hunk 2 above)
Comment (samyak3009) [reply docs/src/pages/components/emptyState/usage.mdx]: reverted it to `showMobile` : true as per our latest discussion.

**Review by anuradha9712 — APPROVED** [review]

**Pull request merged**